html/siptest.html

```diff
@@ -108,10 +108,12 @@ <h3>Demo details</h3>
 								<span class="input-group-addon"><i class="fa fa-phone fa-fw"></i></span>
 								<input class="form-control" type="text" placeholder="SIP URI to call (e.g., sip:1000@example.com)" autocomplete="off" id="peer" onkeypress="return checkEnter(this, event);" />
 							</div>
-							<button class="btn btn-success margin-bottom-sm" autocomplete="off" id="call">Call</button> <input autocomplete="off" id="dovideo" type="checkbox" />Use Video
+							<input class="form-control" type="text" placeholder="any text message" autocomplete="off" id="message" ></input>
+							<button class="btn btn-success margin-bottom-sm" autocomplete="off" id="call">Call</button> <input autocomplete="off" id="dovideo" type="checkbox" />Use Video<br />
+							<button class="btn btn-success margin-bottom-sm" autocomplete="off" id="sendmessage" onclick="return doMessage();" >Message</button>
```

Comment:
Not sure what these are for? We already automatically add buttons for MESSAGE and INFO to calls, e.g.

https://github.com/meetecho/janus-gateway/blob/master/html/siptest.js#L500

Reply:
They are for out-of-dialog messages. There is no way to send messages without making a call first. Do you want me to revert these?